lib/Field/styles.scss

```diff
@@ -69,4 +70,57 @@ $field-data-color: $neutral-base !default;
   position: absolute;
   top: 50%;
   transform: translateY(-50%);
-}
\ No newline at end of file
+  .button--icon-only {
+    margin-bottom: $field-spacing-base/2;
+    &:last-of-type {
+      margin-bottom: 0;
+    }
+  }
+
+}
+
+//statestyles
+
+.is-editor {
```

Comment:
Normally state classes are added when a user performs an action e.g. `is-active` being added after user clicks a button. This feels like it should be a modifier for the `field` component.

Failing that, because the component itself doesn't really do much with the child component we could also just have element styles adding your styles to something like `field__editor`. What do you think? Do we even need to `is-editor` class?

Reply:
Another idea, is to add modifiers to the elements. A really good example would be "instructions".

`field__instructions` & `field__instructions--input`.